spec/feature/logout_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.feature "Logout" do
+  before do
+    reset_application!
+  end
+
+  scenario "Logging out redirects to home page" do
```

Comment:
The app does not have logging in option, so the way it's setup in test is that you cannot log out (it checks for authentication on each request). This can be easily changed, but it will cause either people not being able to log back in once they log out in demo or writing login setup. (which won't be used in prod)